test/st2110-20.sdp

```diff
@@ -0,0 +1,23 @@
+`v=0
+o=- 123456 11 IN IP4 192.168.100.2
+s=Example of a SMPTE ST2110-20 signal
+i=this example is for 720p interlaced video
+t=0 0
+a=recvonly
+a=group:DUP primary secondary
+m=video 50000 RTP/AVP 112
+c=IN IP4 239.100.9.10/32
+a=source-filter:incl IN IP4 239.100.9.10 192.168.100.2
```

Comment:
This line `a=source-filter:incl IN IP4 239.100.9.10 192.168.100.2` is invalid. According to the corresponding [ABNF](https://tools.ietf.org/html/rfc4570#section-3.2.4), there must be a `SP` (space) before `incl`: